jax/experimental/jax2tf/primitives_with_limited_support.md

```diff
@@ -58,7 +58,6 @@ conversion to Tensorflow.
 | nextafter | Missing TF support | Primitive is unimplemented in TF | bfloat16, float16 | CPU, GPU, TPU |
 | population_count | Missing TF support | Primitive is unimplemented in TF | uint32, uint64 | CPU, GPU, TPU |
 | qr | Missing TF support | Primitive is unimplemented in TF; this is a problem only in compiled mode (experimental_compile=True)) | complex128, complex64 | CPU, GPU, TPU |
-| reduce_window_sum | Missing TF support | Primitive is unimplemented in TF | uint16, uint32, uint64 | CPU, GPU, TPU |
```

Comment:
It seems that it was a mistake to remove `StaticArg` around the `init_value` in the `reduce_window` harness maker, as the `reduce_window_{min,max,sum}` specialized paths are not tested anymore, and are thus dropped out of this list.